workbench/connections/workbench-connections.ipynb

```diff
@@ -140,80 +185,42 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "b25303ea-cd23-4958-b08a-c5a52beda700",
    "metadata": {},
-   "outputs": [],
+   "cell_type": "code",
    "source": [
-    "# create a file-like object containing bytes that represent the \"hello world\" string\n",
-    "file_obj = io.BytesIO(b\"hello world\")\n",
-    "\n",
-    "# upload the file-like object to the S3 bucket specified in the data connection\n",
-    "# the name of the \"file\" in S3 is \"hello.txt\"\n",
-    "s3.upload_fileobj(file_obj, bucket_name, Key=\"hello.txt\")"
-   ]
+    "# Upload the dataset.csv file\n",
+    "s3.upload_file(\"./dataset.csv\", bucket_name, \"dataset.csv\")"
+   ],
+   "id": "4f74228da1377bd7",
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "markdown",
    "id": "7b407ad7-14b2-4149-9220-a7188fd3a05c",
    "metadata": {},
-   "source": [
-    "6. List the contents of the bucket specified in the data connection."
-   ]
+   "source": "7. List the contents of the bucket specified in the data connection."
   },
   {
    "cell_type": "code",
-   "execution_count": null,
    "id": "1c0a4879-ed47-4132-8abb-7fcd42cb0af7",
    "metadata": {},
-   "outputs": [],
    "source": [
     "# retrieve the metadata of contents within the bucket\n",
     "objects = s3.list_objects_v2(Bucket=bucket_name)\n",
     "\n",
     "# output the name of each object within the bucket\n",
     "for obj in objects[\"Contents\"]:\n",
     "    print(obj[\"Key\"])"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "63f252dd-3abe-4dd9-ad09-282047b6d97a",
-   "metadata": {},
-   "source": [
-    "> NOTE: Optionally, verify the corresponding S3 bucket for the new `hello.txt` object."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "b9e8cd99-7644-4d02-849b-c30bd744ffaa",
-   "metadata": {},
-   "source": [
-    "7. Download the file from the S3 bucket to a new location."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "0c1ea1b1-dc8e-4027-bc56-75a2ce8084d6",
-   "metadata": {},
+   ],
    "outputs": [],
-   "source": [
-    "s3.download_file(bucket_name, \"hello.txt\", \"new_hello.txt\")"
-   ]
+   "execution_count": null
   },
   {
    "cell_type": "markdown",
-   "id": "c1c591fb-f880-401c-b52f-e719ae83c753",
+   "id": "63f252dd-3abe-4dd9-ad09-282047b6d97a",
    "metadata": {},
-   "source": [
-    "8. The pane to the left displays a new file called `new_hello.txt`.\n",
-    "Open the file and verify that its contents are `hello world`.\n",
-    "\n",
-    "> NOTE: You might need to refresh the file browser by clicking the `Refresh the file browser` button in the file browser pane.\n",
-    "The button displays as a circular arrow."
-   ]
+   "source": "> NOTE: Optionally, verify the corresponding S3 bucket for the new `dataset.csv` object."
```

Comment:
Why not providing them to the URL in the markdown cell so that they can just navigate to the bucket?

Reply:
Since they are listing the bucket contents already, I'll skip login into Minio for brevity.